cli/cmd/cloud_account.go

```diff
@@ -301,6 +301,7 @@ func promptCreateCloudAccount() error {
 				"GCP Audit Log PubSub",
 				"Azure Config",
 				"Azure Activity Log",
+				"Azure Active Directory Activity Log",
```

Comment:
The new name for Azure AD is Microsoft Entra ID. We might want to reflect the name change and say Microsoft Entra ID (formerly Azure Active Directory) in the README.